### DISCUSSION TITLE
Supabase call in getStaticProps

### DISCUSSION THREAD

**Discussion by MrVibe** [supabase/supabase#20345] [General]
Can anyone share an example on how to use Supabse in getStaticProps ? 
Creating a new client does that increase the number of connections ? How can we optimise so that minimal connects are used ?

**Comment by awalias** [comment]
the supabase client doesn't maintain an open connection, it's just a thin wrapper around the REST API, so you can create as many as you need

**Reply by MrVibe** [reply]
awesome thanks.
